locale/en.po

```diff
@@ -3899,6 +3899,9 @@ msgstr "Osmoregulation cost multiplier"
 msgid "OSMOREGULATION_COST_MULTIPLIER_EXPLANATION"
 msgstr "(cost of player species osmoregulation)"
 
+msgid "OTHER"
```

Comment:
The translation key should be more clear, like "OTHER_ITEMS_CATEGORY" or something equally long and clear as I bet many languages will need the additional context for translation".